### PR TITLE
Add media service + nginx location

### PR DESCRIPTION
**Pull request by legendofa**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -24,6 +24,21 @@ server {
         proxy_set_header X-Forwarded-Host $host;
     }
 
+    # Proxy /api/media
+    location /api/media {
+        rewrite ^/api/media/(.*) /$1 break;
+        proxy_set_header Host media-minio:9000;
```
Comment (delvh) [inline nginx.conf]:
```suggestion
        proxy_set_header Host ${MINIO_ENDPOINT};
```

Comment (delvh) [reply nginx.conf]: or
```suggestion
        proxy_set_header Host $host;
```

File: nginx.conf [hunk 2]
```diff
@@ -24,6 +24,21 @@ server {
...
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_connect_timeout 300;
+        # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
```
Comment (delvh) [inline nginx.conf]: `Connection ""`?

Comment (legendofa) [reply nginx.conf]: Not sure what MinIO uses this header for, however this is what they specified in the example: https://min.io/docs/minio/linux/integrations/setup-nginx-proxy-with-minio.html

File: nginx.conf (hunk 1 above)
Comment (legendofa) [reply nginx.conf]: It took me a while to figure out that `$host` does not work in my case, will substitute that with the environment variable

**Review by nk-coding — APPROVED** [review]

File: nginx.conf [hunk 3]
```diff
@@ -24,6 +24,21 @@ server {
...
+    location /api/media {
+        rewrite ^/api/media/(.*) /$1 break;
+        proxy_set_header Host ${MINIO_ENDPOINT};
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_connect_timeout 300;
+        # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
```
Comment (delvh) [inline nginx.conf]:
```suggestion
        proxy_set_header Connection ""; # Seems to be required by MinIO
```

**Review by delvh — APPROVED** [review]

**Pull request merged**